content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst

```diff
@@ -2,99 +2,135 @@
 Add inventory to a new warehouse
 ================================
 
-If a new warehouse is created that has existing inventory in it, the inventory counts should be
-added to Odoo so that the stock listed in the Odoo database reflects what is in the physical
-warehouse. To add inventory to a new warehouse, navigate to :menuselection:`Inventory -->
-Operations --> Inventory Adjustments`, and then click :guilabel:`Create`. The inventory adjustment
-form can then be filled out as follows:
-
-- :guilabel:`Inventory Reference`: the name or code that the inventory adjustment can be referred to
-  by
-- :guilabel:`Locations`: the location(s) where the inventory is stored; include the new warehouse
-  and any locations within it that inventory will be added to
-- :guilabel:`Products`: include all products that will be added to inventory or leave blank to
-  select any product during the next step
-- :guilabel:`Include Exhausted Products`: include products with a quantity of zero; does not affect
-  inventory adjustments for new warehouses since they have no existing inventory
-- :guilabel:`Accounting Date`: the date used by accounting teams for bookkeeping related to the
-  inventory
-- :guilabel:`Company`: the company that owns the inventory; can be set as the user's company or as a
-  customer or vendor
-- :guilabel:`Counted Quantities`: choose whether the counted quantities for products being added
-  should default to stock on hand or zero; does not affect inventory adjustments for new warehouses
-  since they have no existing inventory
-
-.. image:: create_a_second_warehouse/inventory-adjustment-configuration.png
-   :align: center
-   :alt: A filled out form for an inventory adjustment.
+If a new warehouse is created with existing inventory in it, the inventory count should be added to
+Odoo, so the stock listed in the Odoo database reflects the real stock count in the physical
+warehouse.
 
-Once the form is properly configured, click on :guilabel:`Start Inventory` to be taken to the next
-page where products can be added to the inventory adjustment. Add a new product by clicking on
-:guilabel:`Create` and then fill out the product line as follows:
+Inventory can be added to a new warehouse via an inventory adjustment. Navigate to
+:menuselection:`Inventory app --> Operations --> Physical Inventory`, and click :guilabel:`New`.
+Doing so adds a new line at the bottom of the list of inventory adjustments.
 
-- :guilabel:`Product`: the product being added to inventory
-- :guilabel:`Location`: the location where the product is currently stored in the new warehouse;
-  this can be set as the overall warehouse or a location within the warehouse
+Configure the line by filling out the following fields as necessary:
+
+- :guilabel:`Location`: the location where the product is currently stored in the new warehouse.
+  This can be set as the overall warehouse, or a location within the warehouse.
+- :guilabel:`Product`: the product being added to inventory.
```

Comment:
This section on defining the fields in the Inventory adjustment page is really helpful! However, it seems like we might have some duplicate information in our documentation, since the [Inventory Adjustments doc](https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.html) already covers this in great detail. Maybe we could streamline it a bit to avoid redundancy? 

We can either replace this and the following lines with a "seealso" admonition block linking to the Inventory Adjustments page, and/or summarize what we can do on this page.

Reply:
I understand the logic here, but I feel like the process being explained here isn't a bad thing, especially considering it could be part of the workflow for creating a second WH. I did ad a seealso block for that doc, but I'm gonna keep this section as is for now. If I receive the same feedback again, I'll consider changing it